client.go

```diff
@@ -1,20 +1,27 @@
-package redis
+package dpredis
 
 import (
 	"context"
+	"encoding/json"
 	"errors"
-	"github.com/ONSdigital/log.go/log"
 	"time"
 
+	. "github.com/ONSdigital/dp-redis/interfaces"
 	"github.com/ONSdigital/dp-sessions-api/session"
-	goredis "github.com/go-redis/redis"
+	"github.com/ONSdigital/log.go/log"
+	"github.com/go-redis/redis"
 )
 
 var ctx = context.Background()
 
-// Client - structure for the redis client
+// RedisClient - structure for the redis client
```

Comment:
☝️ the `ctx` should be passed into each public method of the client by the calling code.

Reply:
Done